book/src/filters.md

```diff
@@ -78,6 +102,46 @@ Output:
 -  a  -
 ```
 
+### default
+[#default]: #default
+
+```jinja
+{{ variable_or_expression | default(default_value) }}
+{{ variable_or_expression | default(default_value, [[boolean =] true]) }}
+```
+
+This filter works like the Jinja filter of the same name.
+If the second argument is not an boolean `true`, then the filter behaves like [`|defined_or`][#defined_or].
+If it is supplied and `true`, then the filter behaves like [`|assigned_or`][#assigned_or].
```

Comment:
This should maybe describe how this works without having to link to the jinja / the other filters.